src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_4_5_6/SpawnObject.java

```diff
@@ -59,6 +59,12 @@
 				objectdata--;
 				break;
 			}
+			case MINECART: {
+				if (version.isAfterOrEq(ProtocolVersion.MINECRAFT_1_6_1)) {
+					y += 16;
+				}
+				break;
+			}
```

Comment:
This for minecart y location problem in 1.6~1.7.
In 1.6~1.7, minecart passenger looks down position likes 1.5.2.
So I added y 16(0.5)

Reply:
And 1.5.2 need it too.
So if statement will be remove.

Reply:
Remove the if statement then.

Reply:
Ok